spec/support/statistics_test_helper.rb

```diff
@@ -50,6 +50,12 @@ def generate_statistics_test_data
            course_option: course_option_with(level: 'secondary', program_type: 'higher_education_programme', region: 'north_west', subjects: [secondary_subject('Art and design'), secondary_subject('History')]),
            application_form: declined_form)
 
+    withdrawn_form = create(:application_form, :minimum_info, :with_equality_and_diversity_data, sex: 'female', date_of_birth: date_of_birth(years_ago: 80), region_code: :west_midlands, phase: 'apply_1')
```

Comment:
This is fantastic — by definition, I guess we've got all the edge cases now?! Or at least all the ones we and TAD know about. I bet we could make this file a bit more expressive too (later).